app/Http/Controllers/ProfileController.php

```diff
@@ -472,6 +472,15 @@ public function editCampSave(Request $request, $event_id)
         ]);
     }
 
+    // Setup payment for an existing registration
+    public function setupExistingPayment(Request $request, Event $event)
```

Comment:
This naming is not in line with the rest of the system. Why not "payExistingEvent", and please remove the comment, it doesn't add any new context to the code.